documentation/modules/auxiliary/scanner/snmp/snmp_enumusers.md

```diff
@@ -0,0 +1,33 @@
+## Description
+This module queries a range of hosts via SNMP and gathers a list of usernames on the remote system.
+
```

Comment:
There are 2 values, I would call them out and what they map to: https://github.com/rapid7/metasploit-framework/blob/master/modules/auxiliary/scanner/snmp/snmp_enumusers.rb#L36